plugins/uartdmx/UartDmxThread.cpp

```diff
@@ -22,9 +22,11 @@
 #include <math.h>
 #include <unistd.h>
 #include <string>
+#include <algorithm>
 #include "ola/Clock.h"
+#include "ola/Constants.h"
 #include "ola/Logging.h"
-#include "ola/StringUtils.h"
+#include "ola/StringUtils.h
```

Comment:
Fix the include quoting:
```suggestion
#include "ola/StringUtils.h"
```